corehq/apps/hqadmin/templates/hqadmin/superuser_management.html

```diff
@@ -56,4 +56,10 @@ <h2> {{ table_title }}</h2>
       </table>
     </div>
   </div>
+  <style>
```

Comment:
Setting [error_css_class](https://docs.djangoproject.com/en/4.0/ref/forms/api/#django.forms.Form.error_css_class) to `list_unstyled` on the form should give this same effect in a more django-y way.

Reply:
This didn’t seem to work in getting rid of the bullet points and after googling around it looks like `error_css_class` is mainly for highlighting the specific field that raised an error (in this case none because they’re all raised while cleaning data). I also couldn’t find any references to `list_unstyled` in django - were you thinking of the bootstrap class?

Reply:
oops, I meant `list-unstyled` the bootstrap class, and apparently misread the django docs. I was thinking of `MESSAGE_TAGS`. It looks like you can add extra tags when calling `messages.error` or similar functions, but I don't know if/how it's possible to modify the CSS for a single form's errors.